zb-db/src/main/java/io/camunda/zeebe/db/impl/rocksdb/transaction/TransactionalColumnFamily.java

```diff
@@ -47,20 +47,12 @@
     columnFamilyContext = new ColumnFamilyContext(columnFamily.ordinal());
   }
 
-  private void ensureInOpenTransaction(
```

Comment:
:x: You came up with some code conventions, e.g. that `ensureInOpenTransaction` is only called from public methods, not from private methods. Please document these conventions either on class level or on the method level. 

Same for the preference to make `forEachInPrefix` the central method to route to

Reply:
:+1: I added some documentation on both the class and method level.